rmgpy/rmg/main.py

```diff
@@ -1121,12 +1121,27 @@ def run_uncertainty_analysis(self):
                                     time_criteria = ([criteria.time.value], criteria.time.units)
                                     break
                             else:
-                                time_criteria = self.uncertainty['time']
+                                if self.uncertainty['time']:
+                                    time_criteria = ([self.uncertainty['time'].value], self.uncertainty['time'].units)
+                                else:
+                                    raise InputError('If terminationTime was not specified in the reactor options block, it'
+                                                    'must be specified in the uncertainty options block for global uncertainty'
+                                                    'analysis.')
+
+                                    
                             Tlist = ([reaction_system.sens_conditions['T']], 'K')
-                            Plist = ([reaction_system.sens_conditions['P']], 'Pa')
+                            try:
+                                Plist = ([reaction_system.sens_conditions['P']], 'Pa')
+                            except KeyError:
+                                #LiquidReactor
+                                Plist = ([1e8], 'Pa')
+
```

Comment:
A minor comment: Can you add a warning mentioning P = 1e8 Pa is used?

Reply:
@xiaoruiDong Thanks for your review and comment. I've added the warning in logging.info for this.

Reply:
Hao-Wei, is there a reason you choose to use `logging.info` instead of `logging.warning`?